app/src/main/java/org/p2p/wallet/pnl/storage/PnlStorageContract.kt

```diff
@@ -0,0 +1,39 @@
+package org.p2p.wallet.pnl.storage
+
+import java.time.ZonedDateTime
+import org.p2p.core.crypto.Base58String
+import org.p2p.wallet.pnl.models.PnlData
+import org.p2p.wallet.pnl.models.PnlTokenData
+
+interface PnlStorageContract {
+
+    companion object {
+        const val EXPIRE_TIME_MINUTES: Long = 5
+    }
+
+    var total: PnlTokenData
+    var tokens: Map<Base58String, PnlTokenData>
+    var lastUpdateTime: ZonedDateTime?
+
+    fun hasToken(mintAddress: Base58String): Boolean = tokens.containsKey(mintAddress)
```

Comment:
can you put those inside the impl? I am not acquainted  with default methods in interaces 😿